go/pkg/sysdb/coordinator/table_catalog.go

```diff
@@ -218,7 +225,7 @@ func (tc *Catalog) GetAllTenants(ctx context.Context, ts types.Timestamp) ([]*mo
 	return result, nil
 }
 
-func (tc *Catalog) createCollectionImpl(txCtx context.Context, createCollection *model.CreateCollection, ts types.Timestamp) (*model.Collection, bool, error) {
+func (tc *Catalog) createCollectionImpl(txCtx context.Context, createCollection *model.CreateCollection, versionFileName string, ts types.Timestamp) (*model.Collection, bool, error) {
```

Comment:
Is there a possibility that the version file name will be anything other than "" during create collection? If not then suggest not having this parameter in the catalog api rather just setting it to "" when writing to the DB, wdyt?